src/core/Table/Table.tsx

```diff
@@ -373,6 +432,7 @@ export const Table = <
             const rowProps = row.getRowProps({
               className: cx('iui-tables-row', {
                 'iui-tables-row-active': row.isSelected,
+                'iui-tables-expanded': row.isExpanded,
```

Comment:
Does it do anything?

Reply:
Yes, just made a PR for styling changes, will be useful for combination of active and expanded rows. And probably for a different background as well. https://github.com/iTwin/iTwinUI/pull/82